Journalator/Source/SlashCmd.lua

```
@@ -56,13 +52,103 @@ function Journalator.SlashCmd.Debug(...)
+    end
+    -- Get item ID from name (simplified - would need better item lookup)
+    local itemId = 0 -- This would need proper item ID lookup
+    if Journalator.InvestONator.AddItemToPortfolio(portfolioId, itemId, itemName, amountValue) then
+      local formattedAmount = Journalator.InvestONator.FormatGold(amountValue)
+      local msg = "Added " .. itemName .. " with " .. formattedAmount .. " target to portfolio"
+      Journalator.Utilities.Message(msg)
+    else
+      Journalator.Utilities.Message("Failed to add item to portfolio. Check your input.")
+    end
+  elseif command == "list" then
+    local portfolios = Journalator.InvestONator.GetAllPortfolios()
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'Journalator'

(W113)